service-commands/scripts/A

```diff
@@ -91,13 +91,18 @@ def setup(name, service, config, user):
 
     ip = proc.stdout.decode().strip()
     if proc.returncode != 0:
-        input(f"Creating new instance '{name}'. Press Enter to contiue...")
+        input(f"Creating new instance '{name}'. Press Enter to continue...")
         ip = create_instance(
-            name, "ubuntu-os-cloud", "ubuntu-2004-lts", "100", "n2-standard-4"
+            name, "ubuntu-os-cloud", "ubuntu-2004-lts", "100", "n2-custom-12-24576"
         )
 
     host = f"{user}@{ip}"
 
+    print("Waiting for instance to start")
+    returncode = None
+    while returncode != 0:
```

Comment:
should this be `while returncode != None or 0`

Reply:
This line should check for `0` so that we know that the process did not error on line 104

Reply:
oooo yes. makes sense. lgtm!